srcpkgs/rustypaste-cli/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'rustypaste-cli'
+pkgname=rustypaste-cli
+version=0.3.0
+revision=1
+build_style=cargo
+short_desc="CLI tool for rustypaste"
+maintainer="Lukas Jordan <dev@lukasjordan.com>"
+license="MIT"
+homepage="https://github.com/orhun/rustypaste-cli"
+changelog="https://github.com/orhun/rustypaste-cli/blob/master/CHANGELOG.md"
+distfiles="https://github.com/orhun/rustypaste-cli/archive/v${version}.tar.gz"
+checksum=691fd3ddcf2c7c9c17728a304bd4299a35812940902b8b31059902abb49b037e
+
+post_install() {
+	vlicense LICENSE
+	vdoc README.md
+	vman man/rpaste.1
```

Comment:
I wasn't sure wether to include this, because rustypaste-cli never reads from `/etc/`.
But it could be useful to have an example config on the system, which can be copied to the appropiate config dir. Should there be a message after the install to tell the user about this?

Reply:
i used `vsconf`, which is for sample configs. it's in the standard place so no need to tell the user

Reply:
You are right, thanks. I amended the commit